src/components/forms/ClaimForm.svelte

```diff
@@ -130,14 +130,27 @@ const determinePayoutOption = (
   }
   return selectedPayoutOption
 }
+const validateForm = () => {
+  assertHas(lossReason, 'Please select a reason for loss or damage')
+  assertHas(situationDescription, 'Please describe the situation')
+  potentiallyRepairable && assertHas(repairableSelection, 'Please specify if the item is repairable')
+  if (isRepairable) {
+    assertHas(repairEstimateUSD, 'Please enter a repair estimate')
+    assertHas(fairMarketValueUSD, 'Please enter a fair market value')
+  }
+  !(isRepairable || isEvacuation) && assertHas(payoutOption, 'Please select a payout option')
```

Comment:
We also need a payoutOption if it's repairable but too expensive. I _think_ this would do it:
```suggestion
  unrepairableOrTooExpensive && assertHas(payoutOption, 'Please select a payout option')
```